Package.swift

```diff
@@ -1,5 +1,10 @@
+// swift-tools-version:4.2
+
 import PackageDescription
 
 let package = Package(
-  name: "Curry"
+  name: "Curry",
+  products: [.library(name: "Curry", targets: ["Curry"])],
+  dependencies: [],
+  targets: [.target(name: "Curry", dependencies: [], path: ".")]
```

Comment:
```suggestion
  targets: [.target(name: "Curry", dependencies: [], path: "Source")],
  swiftLanguageVersions: [.v4, .v4_2]
```

As all the files are inside `Source` directory, SPM will require us to set the path as `Source`.

Reply:
For reference: https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md#target-format-reference